app/common/helpers/collections.py

```diff
@@ -174,6 +176,23 @@ def submit_answer_for_question(self, question_id: UUID, form: DynamicQuestionFor
         data = _form_data_to_question_type(question, form)
         interfaces.collections.update_collection_data(self.collection, question, data)
 
+    def submit_collection(self, user: "User") -> None:
+        collection_complete = self.status == CollectionStatusEnum.COMPLETED
+        if collection_complete:
+            return
+
+        # TODO: is there a nice way to remove this duplication with the status check?
```

Comment:
Note in the next PR I actually found two uses for this so separated it out into a method there https://github.com/communitiesuk/funding-service/pull/255/files#diff-6f42f8e6a224d41ee10b28d9a4433f0349f53be3adf572a9a3f7faeb01185241R129-R136